ssl/ssl_lib.c

```diff
@@ -6523,7 +6636,12 @@ static int nss_keylog_int(const char *prefix,
     }
     *cursor = '\0';
 
```

Comment:
I think there's some room for improvement of the code that already was in here...